core/urls.py

```diff
@@ -11,4 +11,5 @@
     url(r'^grappelli/', include('grappelli.urls')),
     url(r'^admin/', admin.site.urls),
     url(r'^api$', csrf_exempt(authorization_required(GraphQLView.as_view()))),
+    url(r'^gh$', csrf_exempt(GraphQLView.as_view(graphiql=True))),
```

Comment:
@kola-er we will leave this URL there for developer easy testing, once we are done, we can take it out. Also, notice that I removed the authorization from it.